addon/styles/partials/_uni-dropdown.scss

```diff
@@ -17,6 +17,10 @@
     }
   }
 
+  .uni-dropdown__button--error {
+    border-color: color('feedback', 'error');
```

Comment:
I the lastest discussions about this topic we agreed that for CSS we should keep double quotes but confirm with another reviewer